docs/LocationInfo.md

```diff
@@ -0,0 +1,85 @@
+<!--- SPDX-License-Identifier: Apache-2.0 -->
+
+# Maintain and Use Location Info in onnx-mlir
+
+1. [Summary](#Summary)
+2. [ONNX Model](#ONNX-model)
+3. [MLIR File](#MLIR-file)
+
+## Summary
+Support of Location info propagation in transformation is one of the attractive features of MLIR. onnx-mlir can takes advantage of this feature in compiler transformation, and runtime debugging. This document describes how to maintain and use the location info in onnx-mlir.  program of the In onnx-mlir, there are three types of tests to ensure correctness of implementation:
```

Comment:
Typos: ` program of the In onnx-mlir,` => `In onnx-mlir,`

Reply:
It said there are three types of test, but there are five items below. It's a bit confusing.

Reply:
fixed